packages/tldraw/src/lib/ui.css

```diff
@@ -1496,13 +1484,15 @@
 	touch-action: auto;
 }
 
+/* TODO: refactor shortcuts dialog to make it work in small inline components */
```

Comment:
The keyboard shortcut dialog has various issues at smaller inline-component sizes. I think there's no use in porting those over, so will address separately.